src/analytics/formResponseTracker/formResponseTracker.ts

```diff
@@ -116,4 +120,14 @@ export class FormResponseTracker extends FormTracker {
       ? event.submitter.textContent.trim()
       : "undefined";
   }
+
+  /**
+   * Redacts field value if data is flagged as sensitive
+   *
+   * @param {any} string- Text field input
+   * @return {string} The text field or undefined
+   */
+  redactPII(parameter: any): string {
```

Comment:
This param should be able to be a string. Only place we're using `redactPII` is with the `validateParameter` function which looks like it returns a string from following the chain of function calls back to `stripPIIFromString`

Reply:
Ah yes good spot! I've updated